### ISSUE TITLE
Visual design of the user interface

### ISSUE THREAD

**Issue by nikoheikkila** [pelilegacy/pelilegacy.github.io#1]
### Description
As a visitor, I need to be able to use the site without difficulties and it has to be aesthetically pleasing. All design should be checked against current accessibility standards and best practices.

### Acceptance criteria
- HTML, CSS, and JavaScript are well formatted and working
- Front-end should be as fast as possible (eg. short loading time for assets)
- No accessibility violations are present

### Implementation information
- **Branch:** `feat-DESIGN-1`
- **Depends on:** `master`

### Tasks
- [x] Make post formatting look good
- [x] Create collection and page template for podcasts
- [ ] Add navigation to menu bar
- [x] Design footer to be clear and informative

**Comment by nikoheikkila** [comment]
### First iteration

![image](https://user-images.githubusercontent.com/457034/27926730-1cdcbe10-6292-11e7-892d-e61410d1dea0.png)

***

![image](https://user-images.githubusercontent.com/457034/27926760-3283c77c-6292-11e7-88e4-a5a87f127682.png)
